runners/spark/src/test/java/org/apache/beam/runners/spark/SimpleWordCountTest.java

```diff
@@ -25,24 +25,22 @@
 import org.apache.beam.sdk.coders.StringUtf8Coder;
 import org.apache.beam.sdk.io.TextIO;
 import org.apache.beam.sdk.options.PipelineOptionsFactory;
-import org.apache.beam.sdk.testing.PAssert;
```

Comment:
All the changes here look like an import order change, could probably revert this.